spec/ddtrace/contrib/sidekiq/support/helper.rb

```diff
@@ -25,11 +38,43 @@
 
     Sidekiq::Testing.inline!
   end
+end
 
-  let!(:empty_worker) do
-    stub_const('EmptyWorker', Class.new do
-      include Sidekiq::Worker
-      def perform; end
-    end)
+module SidekiqServerExpectations
```

Comment:
This is super cool!

Reply:
Thanks! It took a number of iterations to finally find the right combination of things that make this work, so hopefully it can be useful going forward!